rust/osm2lanes/src/lib.rs

```diff
@@ -262,11 +239,8 @@ mod tests {
             if !test.skip.is_none() && test.skip.unwrap() {
                 continue;
             }
-            let cfg = Config {
-                driving_side: test.driving_side,
-                inferred_sidewalks: true,
-            };
-            let lanes = get_lane_specs_ltr(&test.tags, &cfg);
+            let lc = Locale::builder().driving_side(test.driving_side).build();
```

Comment:
Style nit: I'd prefer spelling out `locale` or at least not using `lc` as an abbreviation. `lc` to some readers may feel too close to "lane-changing" :)